### PR TITLE
Update Screenshot plugin to have option to automatically take screenshot for PvP kills

### PR DESCRIPTION
**Pull request by jessebake**
This just takes a screenshot whenever a PvP kill message is received.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java [hunk 1]
```diff
@@ -122,4 +122,15 @@ default boolean isScreenshotEnabled()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			keyName = "kills",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java]: Use one tab for indentation within annotations

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java [hunk 2]
```diff
@@ -106,6 +103,13 @@
 	private static final Pattern NUMBER_PATTERN = Pattern.compile("([0-9]+)");
 	private static final Pattern LEVEL_UP_PATTERN = Pattern.compile("Your ([a-zA-Z]+) (?:level is|are)? now (\\d+)\\.");
 
+	private static String[] killMessages = {"into tiny pieces and sat on them", "you have obliterated",
+			"falls before your might", "A humiliating defeat for", "With a crushing blow you", "thinking challenging you",
+	"Can anyone defeat you? Certainly", "was no match for you", "You were clearly a better fighter than", "RIP",
+	"You have defeated", "What an embarrassing performance by", "was no match for your awesomeness"};
+
+	private static final ArrayList<String> KILL_MESSAGES = new ArrayList<>(Arrays.asList(killMessages));
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: remove this

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java [hunk 3]
```diff
@@ -106,6 +103,13 @@
 	private static final Pattern NUMBER_PATTERN = Pattern.compile("([0-9]+)");
 	private static final Pattern LEVEL_UP_PATTERN = Pattern.compile("Your ([a-zA-Z]+) (?:level is|are)? now (\\d+)\\.");
 
+	private static String[] killMessages = {"into tiny pieces and sat on them", "you have obliterated",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: Use ImmutableList

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java [hunk 4]
```diff
@@ -395,6 +406,10 @@ String parseLevelUpWidget(WidgetInfo levelUpLevel)
 		return skillName + "(" + skillLevel + ")";
 	}
 
+
```
Comment (Kamielvf) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: Random newlines?

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java [hunk 5]
```diff
@@ -51,13 +52,8 @@
...
 import javax.inject.Inject;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.ChatMessageType;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.Point;
-import net.runelite.api.events.ChatMessage;
-import net.runelite.api.events.WidgetHiddenChanged;
-import net.runelite.api.events.WidgetLoaded;
+import net.runelite.api.*;
```
Comment (Kamielvf) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: Don't use wildcard imports

**Pull request merged**